chore: update readme (#10)
ZuCommunications/health-check-bundle: README.MD

`/ping` for general health check
`/health-check` for more detailed health check for configured services.

To configure the services to be checked, you can add the following configuration to your `zu_health_check.yaml` file:

```yaml
zu_health_check:
    type:
        smtp: false
        doctrine: false
```

> 

Reviewer: Just noticed that this should remain inside the README.

Author: It is in there twice. I left it in the configuration section

...
```

> To configure the services to be checked, you can add the following configuration to your `zu_health_check.yaml` file at the root of your project:

Reviewer: ```suggestion
To configure the services to be checked, you can add the following configuration to your `zu_health_check.yaml` in the `config` directory found at the root if your project:
```
